integrations/cassandra/src/cassandra_test.go

```diff
@@ -28,29 +30,19 @@ func TestPopulatelMetrics(t *testing.T) {
 		"functionSource": {functionSource, metric.GAUGE},
 	}
 
-	var sample = metric.NewMetricSet("eventType")
-	populateMetrics(&sample, rawMetrics, metricDefinition)
+	s, err := metric.NewSet("eventType", persist.NewInMemoryStore())
+	assert.NoError(t, err)
 
-	if sample["rawMetric1"] != 1 {
-		t.Error()
-	}
-	if sample["rawMetric2"] != 2 {
-		t.Error()
-	}
-	if sample["rawMetric3"] != "foo" {
-		t.Error()
-	}
+	populateMetrics(log.Discard, s, rawMetrics, metricDefinition)
 
-	if sample["unknownMetric"] != nil {
-		t.Error()
-	}
-	if sample["badRawSource"] != nil {
-		t.Error()
-	}
-	if sample["functionSource"] != float64(3) {
-		t.Error()
-	}
+	sample := s.Metrics
 
+	assert.Equal(t, 1.0, sample["rawMetric1"])
+	assert.Equal(t, 2.0, sample["rawMetric2"])
+	assert.Equal(t, "foo", sample["rawMetric3"])
+	assert.Nil(t, sample["unknownMetric"])
+	assert.Nil(t, sample["badRawSource"])
+	assert.Equal(t, 3.0, sample["functionSource"])
```

Comment:
Numeric values are now float values? Why this has changed? Or it's something related to `assert` package?

Reply:
We agreed to use floats for all numeric types: https://github.com/newrelic/infra-integrations-sdk/pull/95/files#diff-83416e4d39b0688a4bec5f4a2c076486R69